src/TestSuite/Fixture/SchemaLoader.php

```diff
@@ -116,8 +116,14 @@ public function loadInternalFile(string $file, string $connectionName = 'test'):
          */
         $connection = ConnectionManager::get($connectionName);
         $connection->disableConstraints(function (Connection $connection) use ($tables): void {
-            foreach ($tables as $table) {
-                $schema = new TableSchema($table['table'], $table['columns']);
+            foreach ($tables as $tableName => $table) {
+                $name = $table['table'] ?? $tableName;
+                if (!is_string($name)) {
+                    throw new InvalidArgumentException(
+                        sprintf('`%s` is not a valid string table name (missing `table` key/value)', $name)
```

Comment:
```suggestion
                        sprintf('`%s` is not a valid table name. Either use a string key for the table definition ( `'articles' => [...]`) or define the `table` key in the table definition.', $name)
```

This might be clearer :shrug: